app/controllers/bookings_controller.rb

```diff
@@ -56,6 +51,18 @@ def scan_ticket
     flash[:notice] = "Ticket does not exist" unless @user_ticket
   end
 
+  def request_refund
+    @booking = Booking.find_by_uniq_id(params[:uniq_id])
+    if @booking.update_attributes(
```

Comment:
What `houdci-bot`  is saying here is that this `request_refund` method can be refactored into 
```

def request_refund 
  @booking = Booking.find_by_uniq_id(params[:uniq_id])

  flash[:notice] = if @booking.update_attributes( refund_requested: true,
                      time_requested: Time.now)
                      "Request for a refund has been sent"
                   else
                      "Request cannot be sent at this time."
                   end

end 

```